proto/githubSearch.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
```

Comment:
This file is straight from the spec Jon sent me, with a few necessary additions at the top